<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MaskSettings</class>
 <widget class="QWidget" name="MaskSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>343</width>
    <height>172</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="Mask">
      <attribute name="title">
       <string>Mask</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <item row="3" column="2">
        <widget class="QSpinBox" name="binning">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>15</number>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Binning:</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QPushButton" name="generated">
         <property name="text">
          <string>Generate</string>
         </property>
        </widget>
       </item>
       <item row="4" column="2">
        <widget class="QPushButton" name="save_txt">
         <property name="text">
          <string>Save as TXT</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QSpinBox" name="rank_of_mask"/>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Mask or function restore:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Transparency of mask:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Rank of mask:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QComboBox" name="mask_or_function">
         <item>
          <property name="text">
           <string>Mask</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Restore function</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QSpinBox" name="transparency_of_mask">
         <property name="minimum">
          <number>-10</number>
         </property>
         <property name="maximum">
          <number>10</number>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QPushButton" name="open_txt">
         <property name="text">
          <string>Open from TXT</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="Convolution">
      <attribute name="title">
       <string>Convolution</string>
      </attribute>
     </widget>
     <widget class="QWidget" name="Backprojection">
      <attribute name="title">
       <string>Backprojection</string>
      </attribute>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
